test/TEST-10-ISSUE-2467/test.sh

```diff
@@ -34,6 +34,7 @@ EOF
 
```

Comment:
Oh man, this test could use a little blurb that would explain what the test does. Took me a bit to understand what it's trying to test.

Your change looks obviously correct, we should probably have done it like this from the beginning. Maybe we could make it even lower, like 3? There's little point in punishing the machine with 200 attempts.

Reply:
That definitely makes sense. And I'll try to add some documentation when I'm at it, as I had to open the original PR to actually understand what's going on.

Reply:
Also, shouldn't we use `StartLimitBurst` instead of `TriggerLimitBurst` in this case, as the original issue was about `StartLimit*` directives not being honoured by socket units?

Reply:
Ah, nevermind, I misunderstood the original issue, `TriggerLimitBurst` is correct.

Reply:
When I wrote the test, I didn't tweak `TriggerLimitBurst` or `StartLimitBurst` to make the test pass because the idea was to make sure that the defaults are good enough even when it's run on slow machines. As that issue shows, they aren't and given that `systemd` is unlikely to be changed, I'd remove the test because it's unclear to me why it should be kept.

Reply:
If I (finally) understand this test (and the original issue) correctly, the main point is to ensure that the `StartLimit*` (and `DefaultStartLimit*`) directives work on non-service units, i.e. they apply to units for which a pre-start condition fails - and that's indeed being tested. The need for a custom `TriggerLimitBurst` is in my opinion a different issue.

Reply:
> The need for a custom TriggerLimitBurst is in my opinion a different issue.

That's exactly what I'm trying to say. If `TriggerLimitBurst` is there, it's no longer https://github.com/systemd/systemd/issues/2467 so at least the test should be renamed.

Reply:
By the way, this test once helped to track down nonsensical logging that slowed down everything considerably. Maybe, it would make sense to try to figure out when the defaults stopped working before trying to make it pass by changing `TriggerLimitBurst`.